### ISSUE TITLE
No check for EDD

### ISSUE THREAD

**Issue by brashrebel** [easydigitaldownloads/edd-braintree#9]
In version 1.1.3 without EDD activated we get:
```
Fatal error: Call to undefined function edd_get_option() in .../test.dev/wp-content/plugins/edd-braintree-master/edd-gateway-braintree.php on line 320
```

**Comment by pippinsplugins** [comment]
Up for tackling this?

**Comment by mintplugins** [comment]
https://github.com/easydigitaldownloads/edd-braintree/pull/10

**Comment by mintplugins** [comment]
Resolved in version 1.1.4
